src/bci_build/package/appcontainers.py

```diff
@@ -724,3 +724,32 @@ def _get_nginx_kwargs(os_version: OsVersion):
     )
     for tomcat_major, os_version in product(_TOMCAT_VERSIONS, ALL_BASE_OS_VERSIONS)
 ]
+
+APACHE2_CONTAINERS = [
+    ApplicationStackContainer(
+        name="apache2",
+        os_version=os_version,
+        package_name="apache2-image",
+        pretty_name="Apache HTTPD Server",
+        is_latest=os_version in CAN_BE_LATEST_OS_VERSION,
+        version="%%apache2_version%%",
+        version_in_uid=False,
+        replacements_via_service=[
+            Replacement(
+                regex_in_build_description="%%apache2_version%%",
+                package_name="apache2",
+                parse_version="minor",
+            )
+        ],
+        license="Apache-2.0",
+        package_list=["apache2"],
+        entrypoint=["httpd", "-DFOREGROUND"],
+        volumes=["/srv/www/htdocs"],
```

Comment:
I don't think that this is a good idea. If I remember it correctly, then this is the documentroot, i.e. where you'll put your html & other static files to be served. However, if you declare this directory to be a volume, then it will be **impossible** to build a container with files in this directory.

This will eliminate the use case of building a container with your static files shipped with the container image.

Reply:
I'm not sure, but the official Docker Hub image does not define a volume, perhaps it is best to not have it.

Reply:
It does not have it for good reasons. If you put `/srv/www/htdocs` into a volume then the following will yield very surprising behavior, namely that `/srv/www/htdocs/` will be **empty** in whatever you just built:
```Dockerfile
from registry.suse.com/suse/apache:2
copy ./static/ /srv/www/htdocs/
```